content/en/serverless/aws_lambda/installation/go.md

```diff
@@ -128,8 +128,10 @@ import (
 	ddlambda "github.com/DataDog/datadog-lambda-go"
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-lambda-go/lambda"
-	httptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"
-	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	httptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http" // 1.x
+  // httptrace "github.com/DataDog/dd-trace-go/contrib/net/http/v2" // 2.x
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer" // 1.x
+  // "github.com/DataDog/dd-trace-go/v2/ddtrace/tracer" // 2.x
```

Comment:
```suggestion
	httptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http" // 1.x
	// httptrace "github.com/DataDog/dd-trace-go/contrib/net/http/v2" // 2.x
	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer" // 1.x
	// "github.com/DataDog/dd-trace-go/v2/ddtrace/tracer" // 2.x
```

Reply:
Ideally we use spaces for this whole snippet, but I can't make the suggestion do that